docs/source/adhoc_instances.rst

```diff
@@ -0,0 +1,37 @@
+=================================
+Running Adhoc Instances in PaaSTA
+=================================
+
+PaaSTA allows users to pre-define and run adhoc instances. These instances can
+be used to run batch jobs, test suites, etc.
```

Comment:
Can you throw on the word "one-off" for people who have heard of heroku:
https://devcenter.heroku.com/articles/one-off-dynos